src/cyclotom.c

```diff
@@ -985,6 +986,9 @@ Obj             AInvCyc (
             exs = EXPOS_CYC(op,len);
             exp = EXPOS_CYC(res,len);
         }
+        else {
+            prd = AInvInt(cfs[i]);
+        }
```

Comment:
I don't understand this part of the change -- why not use `prd = INTOBJ_INT( - INT_INTOBJ(cfs[i]) );` as before? As I see it, the point of this logic here is to use immediate operations (which are guaranteed to not trigGer a GC) explicitly when possible. By invoking `AInvInt` anyway, we are potentially slower, but more worryingly, it is not anymore explicitly clear that no GC can happen here.